RobotFramework_Testsuites/Config/CConfig.py

```diff
@@ -255,7 +255,7 @@ def loadCfg(self):
             ROBFW_AIO_Data.update({key:v})
         oJsonPreprocessor = CJsonPreprocessor(syntax="python", currentCfg=ROBFW_AIO_Data)
         try:
-            oJsonCfgData = oJsonPreprocessor.jsonLoad(self.sTestCfgFile)
+            oJsonCfgData = oJsonPreprocessor.jsonLoad(self.sCalcAbsPath(self, self.sTestCfgFile))
```

Comment:
Hi Son,
in general it is good to use the path functions I put into json-preprocessor.
Just copy them here and replace the once you have currently.
Your functions can have problems with unc-paths and other path constructions.
The once inherited from TML are hopefully stable.
Please also cleanup the CConfig code from old TML codes which are not required here-
Thank you,
 Thomas

Reply:
Hello Thomas,

     I have just checked in JsonPreprocessor code, you already defined and used method __sNormalizePath() to handle jFile (input json file of jsonLoad() method).
     Now, if we again define and use __sNormalizePath() in CConfig of Testsuites-Management to handle json file before call oJsonPreprocessor.jsonLoad(), is it redundant code?
     Can we just simply use os.path.abspath(CConfig.sTestCfgFile) in this case?

Thank you,
Son

Reply:
Hi Son,
yes, please duplicate the function __sNormalizePath(). We need it in jsonpreprocessor and testsuites-management.
This is meaningful because
- it's not meaningful to build a depencency on another repository because of this function.
- this function stabilizes multiple problems which os.path.abspath has.
- this function even allows to use environment variables on both systems, Windows and Linux.
Thank you,
 Thomas